src/main/java/sandbox/SqlHelper.java

```diff
@@ -284,10 +284,37 @@ public static void basicInsert(Connection c, String storageKey,
         }
     }
 
+    public static void insertOrReplace(Connection c,
```

Comment:
Everything in this function is identical to `basicInsert()` besides calling `buildInsertOrReplaceStatement` instead of `buildInsertStatement()`. Would be nice to unify these and just have a boolean arg that controls which of those helpers gets used